pkg/ui/workspaces/cluster-ui/src/databaseDetailsV2/tablesView.tsx

```diff
@@ -100,12 +92,12 @@ const COLUMNS: (TableColumnProps<TableRow> & { sortKey?: TableSortOption })[] =
     },
     {
       title: (
-        <ColumnTitle
-          title={TableColName.NODE_REGIONS}
-          withToolTip={{
-            tooltipText: "Regions/Nodes on which the table's data is stored.",
-          }}
-        />
+        <Tooltip
+          underline
```

Comment:
Is this prop doing anything? I see that `ToolTip` component only accesses `props.noUnderline,` `props.children` and `props.title`

Reply:
Oops, this was leftover from a previous change. Removed.